credentials/apps/badges/admin_forms.py

```diff
@@ -84,6 +84,25 @@ def clean(self):
         return cleaned_data
 
 
+class DataRuleBoolValidationMixin:
+    """
+    Mixin for DataRule form to validate boolean fields.
+    """
+
+    def clean(self):
+        """
+        Validate boolean fields.
+        """
+
+        cleaned_data = super().clean()
+
+        last_key = cleaned_data.get("data_path").split(".")[-1]
+        if "is_" in last_key and cleaned_data.get("value") not in AbstractDataRule.BOOL_VALUES:
```

Comment:
Looks "dirty". Can we reach the parent (requirement/penalty) `event_type` value here?